src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -83,4 +84,13 @@ public static void main(String[] args) {
 
     System.exit(0);
   }
+
+  public static void logSystemInfo() {
```

Comment:
```suggestion
  private static void logSystemInfo() {
```

Reply:
yeah totes

Reply:
On second thought, before I commit to that I need to know how / what should the interactive tiebreak test, test?

Reply:
Are you proposing to have an automated test for interactive in which the test code simulates a user entering a tiebreak selection? Seems maybe excessive. But having a test case available for humans to try out would be reasonable.

Reply:
Agree, I think we need to start distinguishing between "test" files and "sample / example" files. Interactive tiebreaker should definitely be included as sample files in #281, but I think an automated version is much more of an integration / functional test than a unit test, so is out of scope for now.